bezier-rs/lib/src/lib.rs

```diff
@@ -306,3 +330,39 @@ impl Bezier {
 		bezier_starting_at_t1.split(adjusted_t2)[t2_split_side]
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use crate::Bezier;
+	use glam::DVec2;
+
+	fn compare_points(p1: DVec2, p2: DVec2) -> bool {
+		DVec2::new(0.001, 0.001).cmpge(p1 - p2).all()
+	}
+
+	#[test]
+	fn quadratic_from_points() {
+		let p1 = DVec2::new(30., 50.);
+		let p2 = DVec2::new(140., 30.);
+		let p3 = DVec2::new(160., 170.);
+		let bezier = Bezier::quadratic_through_points(p1, p2, p3, 0.5);
```

Comment:
Either call this `bezier1` or consider using shadowing to call them all `bezier`. I also suggest separating these into blocks with blank lines, between the `let p1/2/3` lines and between each of the `let bezier` and `assert!` line groups.